src/dali/plugin/input/rest/coinbase.py

```diff
@@ -70,6 +71,14 @@
 _TYPE: str = "type"
 _UNIT_PRICE: str = "unit_price"
 _USER: str = "user"
+_UPDATED_AT: str = "updated_at"
+_BALANCE: str = "balance"
+
+
+class TransactionList(NamedTuple):
+    In: List[InTransaction]
+    Out: List[OutTransaction]
+    Intra: List[IntraTransaction]
```

Comment:
```suggestion
    in: List[InTransaction]
    out: List[OutTransaction]
    intra: List[IntraTransaction]
```

Reply:
`in` is a python keyword, so if we use it, it will result in a syntax error.

Reply:
Sounds good, never mind then.

Reply:
Or perhaps:
```suggestion
    in_transactions: List[InTransaction]
    out_transactions: List[OutTransaction]
    intra_transactions: List[IntraTransaction]
```

Reply:
How about `{in,out,intra}_list` to keep it shorter?